service/service.go

```diff
@@ -304,6 +289,22 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var operatorFramework *framework.Framework
+	{
+		frameworkConfig := framework.DefaultConfig()
+
+		frameworkConfig.BackOffFactory = framework.DefaultBackOffFactory()
+		frameworkConfig.Informer = newInformer
+		frameworkConfig.InitCtxFunc = initCtxFunc
+		frameworkConfig.Logger = config.Logger
+		frameworkConfig.ResourceRouter = framework.DefaultResourceRouter(resources)
```

Comment:
The TPR should be configured as well to make sure it is created on boot.

Reply:
Done